en/peripherals/companion_computer_peripherals.md

```diff
@@ -44,10 +44,31 @@ Cameras are among the most common form of data collection on a drone whether it
 
 ### Stereo Cameras
 
-Stereo cameras are typically used for depth perception, path planning and SLAM. They are in no way guaranteed to be plug and play with your companion computer.
+Stereo cameras are typically used for depth perception, path planning and SLAM. 
+They are in no way guaranteed to be plug and play with your companion computer.
 
 #### Where To Buy
 * [Intel® RealSense™ Depth Camera D435](https://click.intel.com/intelr-realsensetm-depth-camera-d435.html)
 * [Intel® RealSense™ Depth Camera D415](https://click.intel.com/intelr-realsensetm-depth-camera-d415.html)
 * [DUO MLX ](https://duo3d.com/product/duo-minilx-lv1)
 * [Ironsides](https://www.perceptin.io/products)
+
+
+## Data Telephony (LTE) {#data_telephony}
+
+An LTE USB module can be attached to a companion computer and used to route MAVLink traffic between the flight controller and the Internet.
+
+There is no "standard method" for a ground station and companion to connect over the Internet (i.e. for the GCS to discover the vehicle IP address, and visa versa).
+Depending on your mobile network, it is likely that you won't be able to just directly connect to the remote QGC computer IP over the Internet.
```

Comment:
You said this was due to NAT, but can you explain why? Looking on various forums it looks like mobile networks don't expose a public IP or a fixed/static IP, which I take to mean that you couldn't initiate connection TO the drone from QGC, but I don't see why (if you know the QGC IP) you couldn't connect to it from the vehicle?

Or is this just a matter that you'd have to change the configuration of your intel router if the QGC computer ever moved IP? Not sure that is relevant because again, your VPN is going to have to know the QGC computer IP presumably ...?

Reply:
>  Looking on various forums it looks like mobile networks don't expose a public IP or a fixed/static IP, which I take to mean that you couldn't initiate connection TO the drone from QGC

Exactly

> but I don't see why (if you know the QGC IP) you couldn't connect to it from the vehicle?

Ok, let's dive into networking :) The standard setup of a network (for IPv4) is: you have a local network and within that your GCS machine has a local IP. Between the local network and the internet, there is NAT, which is handled by the router - the router has only one public IP facing to the internet. So there are 2 cases for incoming traffic:
- the target is the public IP of the network, which means the router does the adress translation from public to local IP. Which requires a mapping that is initiated by outgoing traffic. But if you send traffic from the vehicle to the public IP, there is no mapping, so the router will discard the data.
- you might think the vehicle could directly use the local IP of the GCS, but this is not possible - local IP's are reserved for that purpose and not routed through the internet.

Is that clear?

A VPN hides all of this so you don't have to worry.

Reply:
@bkueng . This stuff comes under the set of things "I used to know". Got it again now - thanks very much.